torch/testing/_internal/common_distributed.py

```diff
@@ -330,16 +337,19 @@ def _start_processes(self, proc):
         TEST_SKIPS = test_skips
 
         self.processes = []
+        self.faulthandler_files = []
         for rank in range(int(self.world_size)):
             parent_conn, child_conn = torch.multiprocessing.Pipe()
+            faulthandler_file = tempfile.NamedTemporaryFile()
             process = proc(
                 target=self.__class__._run,
                 name='process ' + str(rank),
-                args=(rank, self._current_test_name(), self.file_name, child_conn))
+                args=(rank, self._current_test_name(), self.file_name, child_conn, faulthandler_file.name))
             process.start()
-            logging.info('Started process {} with pid {}'.format(rank, process.pid))
+            logger.info('Started process {} with pid {}'.format(rank, process.pid))
```

Comment:
nit: can replace str.format with [f-strings](https://docs.python.org/3.6/whatsnew/3.6.html#whatsnew36-pep498). E.g. `f'Started process {rank} with pid {process.pid}'`. Is a bit easier to read and is supposedly faster

Same comment for other strings added in PR.